app/javascript/controllers/action_link_controller.js

```diff
@@ -0,0 +1,31 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static values = { required: { type: Number, default: 0 } };
+
+  // # indicated private attribute or method
```

Comment:
Change `indicated` to `indicates` here and in the selection_controller comment

Reply:
Updated in c8a0cc132ac6188cc6cb2eb2d6a5da02cb34b6ee